src/jit/lsraarmarch.cpp

```diff
@@ -41,14 +41,28 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 //
 void Lowering::TreeNodeInfoInitStoreLoc(GenTreeLclVarCommon* storeLoc)
 {
+    ContainCheckStoreLoc(storeLoc);
     TreeNodeInfo* info = &(storeLoc->gtLsraInfo);
+    GenTree*      op1  = storeLoc->gtGetOp1();
 
-    // Is this the case of var = call where call is returning
-    // a value in multiple return registers?
-    GenTree* op1 = storeLoc->gtGetOp1();
-    if (op1->IsMultiRegCall())
+    info->dstCount = 0;
+#ifdef _TARGET_X86_
+    if (tree->gtGetOp1()->OperGet() == GT_LONG)
     {
-        // backend expects to see this case only for store lclvar.
+        assert(tree->gtGetOp1()->isContained());
```

Comment:
Both before and after fixing this part, I've observed that all tests in `JIT/CodeGenBringUpTests` failed on Rpi3 with ryujit/arm32
Following assertion observed when running with `COMPlus_AltJit=*  COMPlus_AltJitAssertOnNYI=0`.

```
               Assert failure(PID 15977 [0x00003e69], Thread: 15977 [0x3e69]): Assertion failed '((consume == 0) && (produce == 0)) || (ComputeAvailableSrcCount(tree) == consume)' in 'System.AppDomainSetup:SetupDefaults(ref,bool):this' (IL size 84)

                   File: /opt/code/github/hqueue/coreclr/src/jit/lsra.cpp Line: 3620
                   Image: /mnt/albireo/mywork/device/rpi3/unittests/coreoverlays/coreoverlay.test20170710.debugging/corerun

```